test/remote/recovery_email_resend_code_tests.js

```diff
@@ -203,6 +203,7 @@ describe('remote recovery email resend code', function () {
         assert.equal(emailData.headers['x-template-name'], 'verifyPrimaryEmail', 'correct template set')
         const code = emailData.headers['x-verify-code']
         assert.ok(code, 'code set')
+        assert.equal(emailData.html.indexOf('IP address') > -1, true, 'contains ip address')
```

Comment:
Other remote test that use location data, only assert the IP address. I believe that this is because geoDB fails to find a proper location for localhost and returns null, which causes the template not to render the location. FWIW, I tested this by manually setting the location data and checking the email it generated.

<img width="383" alt="screen shot 2018-03-13 at 12 02 11 pm" src="https://user-images.githubusercontent.com/1295288/37353772-67d2d36c-26b6-11e8-9ea5-c0d38a7f2472.png">